src/pages/Explore/index.tsx

```diff
@@ -0,0 +1,196 @@
+import { Trans } from '@lingui/macro'
+import { BrowserEvent, InterfacePageName, SharedEventName } from '@uniswap/analytics-events'
+import { TraceEvent } from 'analytics'
+import { Trace } from 'analytics'
+import { AutoRow } from 'components/Row'
+import { TabbedNav } from 'components/TabbedNav'
+import { MAX_WIDTH_MEDIA_BREAKPOINT, MEDIUM_MEDIA_BREAKPOINT } from 'components/Tokens/constants'
+import { filterStringAtom } from 'components/Tokens/state'
+import NetworkFilter from 'components/Tokens/TokenTable/NetworkFilter'
+import SearchBar from 'components/Tokens/TokenTable/SearchBar'
+import TimeSelector from 'components/Tokens/TokenTable/TimeSelector'
+import TokenTable from 'components/Tokens/TokenTable/TokenTable'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { useInfoExplorePageEnabled } from 'featureFlags/flags/infoExplore'
+import { useResetAtom } from 'jotai/utils'
+import { useEffect, useState } from 'react'
+import { useLocation } from 'react-router-dom'
+import styled from 'styled-components'
+import { ThemedText } from 'theme/components'
+
+const ExploreContainer = styled.div`
+  width: 100%;
+  min-width: 320px;
+  padding: 68px 12px 0px;
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.md}px`}) {
+    padding-top: 48px;
+  }
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.sm}px`}) {
+    padding-top: 20px;
+  }
+`
+const TitleContainer = styled.div`
+  margin-bottom: 32px;
+  max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+`
+const Nav = styled(AutoRow)`
```

Comment:
AutoRow can handle inline gap to reduce the amount of styled components defined
`<AutoRow gap="20px" data-testid="explore-navbar">`